gapic/api/artman_bigtable.yaml

```diff
@@ -1,6 +1,7 @@
 common:
   api_name: bigtable
   api_version: v2
+  package_type: grpc_client
```

Comment:
I have thought about using default but it seems like we do not have such a default yaml (it seems overkill to create a default config file just for this one field).
On the other hand, does Artman tasks allows default values for a field? If I do not add those fields for every API, Artman task just gives me a runtime exception of missing requirements.